feat: add @a_lasselle delegate profile (#52)
unik-name/unikname-network-delegates-website: docs/delegates/9a426c47453de112ea04a2e0114acfc8613aeafe67dd4ed765c75a5278e1dd3f/README.md

---
twitter: AdrienLASSELLE
github: a-lasselle
discord: a_lasselle

> uns forum: a_lasselle

Reviewer: ```suggestion
forum: a_lasselle
```

email: adrien@lasselle.site
---

## Summary

Hi folks !

My name is Adrien, I'm a developer from Nantes City in France.

> I'm a delegate since the very beginning of the UNS network.

Reviewer: ```suggestion
I'm a delegate since the very beginning of the Unikname.network (formerly UNS network).
```

I'm passionate in blockchain and IT. With a friend we have create a talk about it and Duniter blockchain framework, you can view it : [https://www.youtube.com/watch?v=ewdqIoMGT5U](https://www.youtube.com/watch?v=ewdqIoMGT5U)

> With some UNS core team members, we manage a meetup about Blockchain and crypots in Nantes, more informations here : [https://blockchainsociete.org/](https://blockchainsociete.org/)

Reviewer: ```suggestion
With some Unikname.network core team members, we manage a meetup about Blockchain and crypots in Nantes, more informations here : [https://blockchainsociete.org/](https://blockchainsociete.org/)
```
